src/Planners/src/TimeVaryingDCMPlanner.cpp

```diff
@@ -339,13 +339,13 @@ struct TimeVaryingDCMPlanner::Impl
 
         // compute the convex hull only once
         int columnIndex = 0;
-        iDynTree::Position point;
+        Eigen::Vector3d point;
         for (const auto& [contactName, activeContact] : contactPhase.activeContacts)
         {
             for (const auto& footCorner : this->optiSettings.footCorners)
             {
-                point = activeContact->pose * footCorner;
-                points.col(columnIndex) = iDynTree::toEigen(point).head(numberOfCoordinates);
+                point = activeContact->pose.isometry() * footCorner;
```

Comment:
You could also use the act() method. It must be doing the same thing.

Reply:
Done https://github.com/dic-iit/bipedal-locomotion-framework/pull/82/commits/74928a3dbf8c3bce47e90b8b87e9e4c11735d161